opentelemetry-api/pyproject.toml

```diff
@@ -29,7 +29,7 @@ dependencies = [
     "setuptools >= 16.0",
     # FIXME This should be able to be removed after 3.12 is released if there is a reliable API
     # in importlib.metadata.
-    "importlib-metadata ~= 6.0.0",
+    "importlib-metadata ~= 6.0",
```

Comment:
Can I ask what's the difference?

Reply:
>~= 6.0.0
> \>= 6.0.0, == 6.0.*

See https://peps.python.org/pep-0440/#compatible-release